jenkins/Jenkinsfile

```diff
@@ -106,17 +106,17 @@ pipeline {
                         checkout scm
                         sh 'docker login -u "$CANDLEPIN_QUAY_BOT_USR" -p "$CANDLEPIN_QUAY_BOT_PSW" quay.io'
                         sh 'python jenkins/check_pr_branch.py $CHANGE_ID'
-                        sh 'docker logout quay.io'
+                        // sh 'docker logout quay.io'
```

Comment:
@Lorquas Is commenting this out something we should do? Do you think it is responsible for the 'unauthorized' errors we sometimes see when trying to pull images?

Reply:
@nikosmoum   yes i think this logout/login race condition is causing these errors, one test will log out right before another test needs to perform a pull.  I don't see the harm in just removing this line, i think you can just continuously log in over and over to refresh the credential.

Reply:
@Lorquas Logging out changes are already merged into the master branch (PR https://github.com/candlepin/candlepin/pull/3070). 
On rebasing this PR, you will no longer see the logout code.